android/src/main/java/io/sentry/react/RNSentryModuleImpl.java

```diff
@@ -294,6 +294,9 @@ public void initNativeSdk(final ReadableMap rnOptions, Promise promise) {
             }
         });
 
+        Sentry.getCurrentHub().getOptions().getReplayController()
+                .setBreadcrumbConverter(new RNSentryReplayBreadcrumbConverter());
+
```

Comment:
Can we call this inside of the `Sentry.init()` call above? Or is there some side effect due to which it would not work?

Reply:
The replay controller is only available after init() AFAIK.

@romtsn

Reply:
Got it thank you. Let's wait for Roman for confirmation.

Reply:
it should be fine to call it inside `init` as well (`options.getReplayController().setBreacrumbConverter()`), because we instantiate the ReplayIntegration before calling the user's configuration block. But please test it out, I might have missed something

Reply:
Works fine inside init.